### ISSUE TITLE
Feature request: Sorting - av1 and mp4. or a free text option

### ISSUE THREAD

**Issue by Greasygrease5** [synclerd/syncler#99]
more options in sorting results would be helpful-specifically, for me, av1 and mp4.  or a free text option?  thanks for considering!

**Comment by synclerd** [comment]
Do you mean filtering options?

**Comment by Greasygrease5** [comment]
Yes, I guess I was thinking filtering rather than sorting-the badge section where I can filter out cam, or DV.  Having the option to filter out av1 woukd be nice as my devices won't play them.  I know I can long click and take a look at the details of a file, but my family likes the autoplay option.  Just a thought.
Thanks for considering, been enjoying the app for a couole years now and look forward to the upcoming update 👍👍

Mike Blodgett
________________________________

**Comment by LoganWolverineX** [comment]
closing GH item.  if needs to be revisited, GH item can be re-opened or new GH item can be created with new details provided.  i believe v2.x has the ability to do this now with regex
